exoctk/contam_visibility/miniTools.py

```diff
@@ -0,0 +1,363 @@
+""" The contamVerify mini tool will be a companion to ExoCTK's Contamination
+Overlap tool, as it will visualize the Contaminaton Bokeh plots on the website.
+
+Functions are:
+plotTemps    - Plots the temperatures of stars according to color.
+traceLengths - Fine-tunes the trace lengths in the plot.
+contamVerify - The main mini tool. Outputs a .pdf file with one or more figures
+               showing the FOV in the science frame according to the input
+               Aperture Position Angle(s) it is fed.
+
+Author(s)
+---------
+Jennifer V. Medina, 2020
+"""
+import os
+import matplotlib.pyplot as plt
+import numpy as np
+import pysiaf
+
+from astropy.io import fits
+from astroquery.irsa import Irsa
+import astropy.coordinates as crd
+import astropy.units as u
+from scipy.io import readsav
+from matplotlib import cm
+from matplotlib.backends.backend_pdf import FigureCanvasPdf, PdfPages
+from matplotlib.figure import Figure
+
+EXOCTK_DATA = os.environ.get('EXOCTK_DATA')
+if not EXOCTK_DATA:
+    print(
+        'WARNING: The $EXOCTK_DATA environment variable is not set. Contamination overlap will not work. Please set the '
+        'value of this variable to point to the location of the exoctk_data '
+        'download folder.  Users may retreive this folder by clicking the '
+        '"ExoCTK Data Download" button on the ExoCTK website, or by using '
+        'the exoctk.utils.download_exoctk_data() function.')
+    TRACES_PATH = None
+
+TRACES_PATH = os.path.join(EXOCTK_DATA, 'exoctk_contam', 'traces')
+
+
+def plotTemps(TEMPS, allRA, allDEC):
```

Comment:
I think PEP8 prefers that function names use underscores (e.g. `plot_temps`) instead of CamelCase.  Usually CamelCase is used for class definitions.